View3D/Services/WsModelGeneratorService.cs

```diff
@@ -144,6 +217,13 @@ string LoadMaterialTemplate()
             var result = reader.ReadToEnd();
             return result;
         }
+        string LoadMaterialTemplateWh2()
```

Comment:
I think the only difference in code should be the materials. Maybe have two MaterialGenerator classes, and instanciate the correct one based on an input enum (GameType, cant remember what is actually called) Throw exception if game not supported

Reply:
OK, noted. I will push another changes.